tests/e2e/addons/TA_fiction/default/transforms.conf

```diff
@@ -22,6 +22,23 @@ SOURCE_KEY = event_id
 DELIMS = "="
 FIELDS = server_contact_mode, dest
 
+# Component tested: REPORT, DELIM-FIELDS-SOURCE_KEY
+# Scenario:# Similar to the above scenario
+## ## server-contact-mode should be searched as server_contact_mode as CLEAN_KEYS = true by default
```

Comment:
CLEAN_KEYS is set to false here, so how should this field be looked for? Just description required small adjustment.

Reply:
done